arcaflow_plugin_horreum_client/horreum_client_plugin.py

```diff
@@ -61,9 +71,19 @@ def horreum_client(
             verify=params.tls_verify,
         )
 
-    except (requests.ConnectionError, requests.HTTPError, requests.Timeout):
+    except requests.exceptions.SSLError as e:
+        return "error", ErrorOutput(
+            f"Unable to establish SSL connection with the Horreum server: {e}"
+        )
+
+    except (requests.ConnectionError, requests.Timeout) as e:
+        return "error", ErrorOutput(
+            f"Error communicating with the Horreum server: {e}"
+        )
+
+    except requests.HTTPError:
         return "error", ErrorOutput(
-            f"Error communicating with the Horreum server: {auth_return.text}"
+            f"Error returned from the Horreum server: {auth_return.text}"
         )
```

Comment:
I'm not sure what [Dave's comment](https://github.com/arcalot/arcaflow-plugin-horreum-client/pull/3#discussion_r1420365391) refers to exactly, but `auth_return` was set by the Keycloak request, and it was _**not**_ returned by the Horreum server, as far as I can tell...so, _something_ is dreadfully wrong with this message....